packages/stark-rbac/src/modules/authorization/entities/state-permissions.entity.intf.ts

```diff
@@ -0,0 +1,40 @@
+import { StarkStateRedirection, StarkStateRedirectionFn } from "./state-redirection.intf";
+
+/**
+ * Describes the content of the 'permissions' object to be set inside the 'data' property of a state configuration
+ *
+ * ```typescript
+ * export const APP_STATES: Ng2StateDeclaration[] = [
```

Comment:
As suggested yesterday, could you create an interface `StarkRBACNg2StateDeclaration` which provides our down definitions ? 😊 
```typescript
import { Ng2StateDeclaration } from "@uirouter/angular";
import { StarkRBACStatePermissions } from "@nationalbankbelgium/stark-rbac";

export type StarkRBACNg2StateDeclaration = Ng2StateDeclaration & {
  data: {
    permissions: StarkRBACStatePermissions
  }
};
```

And then you'll have to auto-completion when you set `permissions`, otherwise it stays the same StateDeclaration as the original 😊